### PR TITLE
Add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - `feature` for changes that change or add new functionality
+    - `refactor` for changes that does not change program behaviour
+- Append tag with JIRA task and description, i.e.:
+    - `feature/MYS-123-favorites`
+    - `bugfix/MYS-100-missing-logs`
+    - `hotfix/crash-with-unknown-names`
+    
+### Communication
+
+We are all humans, so have in mind that we can make mistakes, be subjective and we have a feelings.
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: We're all people, so have in mind that we can make mistakes, be subjective and have a feelings.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+When communicating in PRs, we should:
+
+- Be subjective when possible - i.e. using statistics or external resources instead of yourself opinions
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: Be objective when possible..

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- Be open to different opinions - everybody have different mindset, and by merging different mindsets we can get
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: Be open to different opinions - everyone can have a different mindset and by merging different mindsets we can get to a group solution

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+We are all humans, so have in mind that we can make mistakes, be subjective and we have a feelings.
+When communicating in PRs, we should:
+
+- Be subjective when possible - i.e. using statistics or external resources instead of yourself opinions
+- Be open to different opinions - everybody have different mindset, and by merging different mindsets we can get 
+- Consider talking in real life (or even pair-programming) when communication online gets tricky
+
+### Making PR
+
+When making a PR, make sure you've prepared it well to safe time and energy of yours and you teammates:
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: When making a PR, make sure you've prepared it well in order to save your own time as well as time and energy of your teammates:

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+- Make sure CI is passing
+- Before making PR, review changes as you were a reviewer - see *Reviewing PR* section
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: Before making a PR, review changes as you were the reviewer

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- (Optional) Add a PR description if changes are not obvious
+
+### Getting feedback
+
+Not many people think reviewing PR is fun, so getting a feedback is like getting a 🎁.
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: Not many people think reviewing PRs is fun, but getting feedback is like..

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+Code of Conduct
+===============
+
+This document describes how we collaborate developing Mysterion.
+
+Main collaboration goals:
+- Fast features delivery
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: Fast feature delivery

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+When making a PR, make sure you've prepared it well to safe time and energy of yours and you teammates:
+
+- Make sure CI is passing
+- Before making PR, review changes as you were a reviewer - see *Reviewing PR* section
+- (Optional) Add a PR description if changes are not obvious
+
+### Getting feedback
+
+Not many people think reviewing PR is fun, so getting a feedback is like getting a 🎁.
+If you disagree with comment, try discussing it and understanding why opinions differ and how to make both sides happy.
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: If you disagree with a comment, try discussing it and understanding why opinions differ and how to make all sides happy.

File: CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+Consider asking opinions of others.
+
+### Reviewing PR
+
+When reviewing PR, try focusing to most important things first. More about that in *Coding values* section.
```
Comment (ignasbernotas) [inline CODE_OF_CONDUCT.md]: When reviewing PRs, try focusing on the most important things first..

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (donce) [reply CODE_OF_CONDUCT.md]: Why **but** instead of **so**? I had in mind that because reviewing PRs is hard, we should appreciate it.

**Pull request merged**